ekf_localization:
    ros__parameters:
      reference_frame_id: "map"
      robot_frame_id: "base_link"
      gnss_pose_topic: "gnss_pose"
      imu_topic: "imu"
      odom_topic: "odom"
      pub_period: 10
      var_imu_w: 0.01
      var_imu_acc: 0.01
      var_gnss: 0.1
      var_odom: 0.1
      use_gnss: true
      use_odom: false